[pdal] Build failue

Package: pdal[core]:arm64-osx -> 2.4.3#2

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 1e68748a7 2023-04-17 (2 hours ago)

**To Reproduce**

`vcpkg x-set-installed pdal[core]`

**Failure logs**

```
-- Note: pdal only supports dynamic library linkage. Building dynamic library.
-- Using cached PDAL-PDAL-2.4.3.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/PDAL-PDAL-2.4.3.tar.gz
-- Applying patch fix-dependency.patch
-- Applying patch fix-unix-compiler-options.patch
-- Applying patch fix-find-library-suffix.patch
-- Applying patch no-pkgconfig-requires.patch
-- Applying patch no-rpath.patch
-- Applying patch fix-gcc8-compatibility.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/nanoflann/nanoflann.hpp
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/nlohmann/nlohmann/json.hpp
-- Installing: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/pdal/json_fwd.hpp
-- Found external ninja('1.11.1').
-- Configuring arm64-osx
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/pdal/portfile.cmake:69 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/install-arm64-osx-dbg-out.log</summary>

```
[1/271] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DH5_BUILT_AS_STATIC_LIB -DUNIX -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -Wno-unknown-warning-option -fPIC -std=c++17 -MD -MT vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/Factor.cpp.o -MF vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/Factor.cpp.o.d -o vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/Factor.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/kazhdan/Factor.cpp
[2/271] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DH5_BUILT_AS_STATIC_LIB -DUNIX -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -Wno-unknown-warning-option -fPIC -std=c++17 -MD -MT vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/MarchingCubes.cpp.o -MF vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/MarchingCubes.cpp.o.d -o vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/MarchingCubes.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/kazhdan/MarchingCubes.cpp
[3/271] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DH5_BUILT_AS_STATIC_LIB -DUNIX -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -Wno-unknown-warning-option -fPIC -std=c++17 -MD -MT vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/Geometry.cpp.o -MF vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/Geometry.cpp.o.d -o vendor/kazhdan/CMakeFiles/pdal_kazhdan.dir/Geometry.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/kazhdan/Geometry.cpp
...
Skipped 265 lines
...

[261/271] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DH5_BUILT_AS_STATIC_LIB -DPROJ_DLL="" -DUNIX -Dpdalcpp_EXPORTS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/eigen -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/nlohmann -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -Wno-unknown-warning-option -std=c++17 -MD -MT CMakeFiles/pdalcpp.dir/pdal/private/gdal/Raster.cpp.o -MF CMakeFiles/pdalcpp.dir/pdal/private/gdal/Raster.cpp.o.d -o CMakeFiles/pdalcpp.dir/pdal/private/gdal/Raster.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/pdal/private/gdal/Raster.cpp
[262/271] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DH5_BUILT_AS_STATIC_LIB -DPDAL_DLL_EXPORT -DUNIX -Dpdal_plugin_kernel_fauxplugin_EXPORTS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -Wno-unknown-warning-option -std=c++17 -MD -MT plugins/faux/CMakeFiles/pdal_plugin_kernel_fauxplugin.dir/kernel/FauxPluginKernel.cpp.o -MF plugins/faux/CMakeFiles/pdal_plugin_kernel_fauxplugin.dir/kernel/FauxPluginKernel.cpp.o.d -o plugins/faux/CMakeFiles/pdal_plugin_kernel_fauxplugin.dir/kernel/FauxPluginKernel.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/plugins/faux/kernel/FauxPluginKernel.cpp
[263/271] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DH5_BUILT_AS_STATIC_LIB -DPROJ_DLL="" -DUNIX -Dpdalcpp_EXPORTS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/eigen -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/nlohmann -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -Wno-unknown-warning-option -std=c++17 -MD -MT CMakeFiles/pdalcpp.dir/pdal/private/gdal/GDALUtils.cpp.o -MF CMakeFiles/pdalcpp.dir/pdal/private/gdal/GDALUtils.cpp.o.d -o CMakeFiles/pdalcpp.dir/pdal/private/gdal/GDALUtils.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/pdal/private/gdal/GDALUtils.cpp
[264/271] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DH5_BUILT_AS_STATIC_LIB -DPROJ_DLL="" -DUNIX -Dpdalcpp_EXPORTS -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/eigen -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/nlohmann -I/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -Wno-unknown-warning-option -std=c++17 -MD -MT CMakeFiles/pdalcpp.dir/pdal/private/MathUtils.cpp.o -MF CMakeFiles/pdalcpp.dir/pdal/private/MathUtils.cpp.o.d -o CMakeFiles/pdalcpp.dir/pdal/private/MathUtils.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/pdal/private/MathUtils.cpp
[265/271] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DH5_BUILT_AS_STATIC_LIB -DUNIX -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/vendor/nlohmann -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -Wextra -Wpointer-arith -Wcast-align -Wcast-qual -Wno-error=parentheses -Wno-error=cast-qual -Wredundant-decls -Wno-unused-parameter -Wno-unused-variable -Wno-long-long -Wno-unknown-pragmas -Wno-deprecated-declarations -Wno-unknown-warning-option -std=c++17 -MD -MT apps/CMakeFiles/pdal.dir/pdal.cpp.o -MF apps/CMakeFiles/pdal.dir/pdal.cpp.o.d -o apps/CMakeFiles/pdal.dir/pdal.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/src/2.4.3-0afe756b12.clean/apps/pdal.cpp
[266/271] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -dynamiclib -Wl,-headerpad_max_install_names -compatibility_version 14.0.0 -current_version 14.3.0 -o lib/libpdalcpp.14.3.0.dylib -install_name @rpath/libpdalcpp.14.dylib CMakeFiles/pdalcpp.dir/filters/ApproximateCoplanarFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/AssignFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/CSFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ChipperFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ClusterFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ColorinterpFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ColorizationFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/CovarianceFeaturesFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/CropFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DBSCANFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DEMFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DecimationFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DelaunayFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DividerFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ELMFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/EigenvaluesFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/EstimateRankFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/FaceRasterFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/FarthestPointSamplingFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/FerryFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/GpsTimeConvert.cpp.o CMakeFiles/pdalcpp.dir/filters/GreedyProjection.cpp.o CMakeFiles/pdalcpp.dir/filters/GroupByFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HagDelaunayFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HagDemFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HagNnFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HeadFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HexBinFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/IQRFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/InfoFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/IterativeClosestPoint.cpp.o CMakeFiles/pdalcpp.dir/filters/LOFFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/LiTreeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/LloydKMeansFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/LocateFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MADFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MergeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MiniballFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MongoExpressionFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MortonOrderFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/NNDistanceFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/NeighborClassifierFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/NormalFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/OptimalNeighborhoodFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/OutlierFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/OverlayFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/PMFFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/PlaneFitFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/PoissonFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ProjPipelineFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/RadialDensityFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/RandomizeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/RangeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ReciprocityFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/RelaxationDartThrowing.cpp.o CMakeFiles/pdalcpp.dir/filters/ReprojectionFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ReturnsFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SMRFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SampleFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SeparateScanLineFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ShellFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SkewnessBalancingFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SortFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SplitterFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/StatsFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/StreamCallbackFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/TailFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/TransformationFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/VoxelCenterNearestNeighborFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/VoxelCentroidNearestNeighborFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/VoxelDownsizeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ZsmoothFilter.cpp.o CMakeFiles/pdalcpp.dir/io/BpfCompressor.cpp.o CMakeFiles/pdalcpp.dir/io/BpfHeader.cpp.o CMakeFiles/pdalcpp.dir/io/BpfReader.cpp.o CMakeFiles/pdalcpp.dir/io/BpfWriter.cpp.o CMakeFiles/pdalcpp.dir/io/CopcReader.cpp.o CMakeFiles/pdalcpp.dir/io/CopcWriter.cpp.o CMakeFiles/pdalcpp.dir/io/EptAddonWriter.cpp.o CMakeFiles/pdalcpp.dir/io/EptReader.cpp.o CMakeFiles/pdalcpp.dir/io/FauxReader.cpp.o CMakeFiles/pdalcpp.dir/io/GDALReader.cpp.o CMakeFiles/pdalcpp.dir/io/GDALWriter.cpp.o CMakeFiles/pdalcpp.dir/io/GltfWriter.cpp.o CMakeFiles/pdalcpp.dir/io/Ilvis2MetadataReader.cpp.o CMakeFiles/pdalcpp.dir/io/Ilvis2Reader.cpp.o CMakeFiles/pdalcpp.dir/io/LasHeader.cpp.o CMakeFiles/pdalcpp.dir/io/LasReader.cpp.o CMakeFiles/pdalcpp.dir/io/LasVLR.cpp.o CMakeFiles/pdalcpp.dir/io/LasWriter.cpp.o CMakeFiles/pdalcpp.dir/io/MemoryViewReader.cpp.o CMakeFiles/pdalcpp.dir/io/NullWriter.cpp.o CMakeFiles/pdalcpp.dir/io/OGRWriter.cpp.o CMakeFiles/pdalcpp.dir/io/ObjReader.cpp.o CMakeFiles/pdalcpp.dir/io/OptechReader.cpp.o CMakeFiles/pdalcpp.dir/io/PcdHeader.cpp.o CMakeFiles/pdalcpp.dir/io/PcdReader.cpp.o CMakeFiles/pdalcpp.dir/io/PcdWriter.cpp.o CMakeFiles/pdalcpp.dir/io/PlyReader.cpp.o CMakeFiles/pdalcpp.dir/io/PlyWriter.cpp.o CMakeFiles/pdalcpp.dir/io/PtsReader.cpp.o CMakeFiles/pdalcpp.dir/io/QfitReader.cpp.o CMakeFiles/pdalcpp.dir/io/RasterWriter.cpp.o CMakeFiles/pdalcpp.dir/io/SbetReader.cpp.o CMakeFiles/pdalcpp.dir/io/SbetSmrmsgReader.cpp.o CMakeFiles/pdalcpp.dir/io/SbetWriter.cpp.o CMakeFiles/pdalcpp.dir/io/TIndexReader.cpp.o CMakeFiles/pdalcpp.dir/io/TerrasolidReader.cpp.o CMakeFiles/pdalcpp.dir/io/TextReader.cpp.o CMakeFiles/pdalcpp.dir/io/TextWriter.cpp.o CMakeFiles/pdalcpp.dir/kernels/ChamferKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/DeltaKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/DensityKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/EvalKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/GroundKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/HausdorffKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/InfoKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/MergeKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/PipelineKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/RandomKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/SortKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/SplitKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/TIndexKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/TileKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/TranslateKernel.cpp.o CMakeFiles/pdalcpp.dir/pdal/ColumnPointTable.cpp.o CMakeFiles/pdalcpp.dir/pdal/DbReader.cpp.o CMakeFiles/pdalcpp.dir/pdal/DbWriter.cpp.o CMakeFiles/pdalcpp.dir/pdal/DynamicLibrary.cpp.o CMakeFiles/pdalcpp.dir/pdal/Filter.cpp.o CMakeFiles/pdalcpp.dir/pdal/Geometry.cpp.o CMakeFiles/pdalcpp.dir/pdal/KDIndex.cpp.o CMakeFiles/pdalcpp.dir/pdal/Kernel.cpp.o CMakeFiles/pdalcpp.dir/pdal/Log.cpp.o CMakeFiles/pdalcpp.dir/pdal/Metadata.cpp.o CMakeFiles/pdalcpp.dir/pdal/Options.cpp.o CMakeFiles/pdalcpp.dir/pdal/PDALUtils.cpp.o CMakeFiles/pdalcpp.dir/pdal/PipelineExecutor.cpp.o CMakeFiles/pdalcpp.dir/pdal/PipelineManager.cpp.o CMakeFiles/pdalcpp.dir/pdal/PipelineReaderJSON.cpp.o CMakeFiles/pdalcpp.dir/pdal/PipelineWriter.cpp.o CMakeFiles/pdalcpp.dir/pdal/PluginDirectory.cpp.o CMakeFiles/pdalcpp.dir/pdal/PluginManager.cpp.o CMakeFiles/pdalcpp.dir/pdal/PointLayout.cpp.o CMakeFiles/pdalcpp.dir/pdal/PointTable.cpp.o CMakeFiles/pdalcpp.dir/pdal/PointView.cpp.o CMakeFiles/pdalcpp.dir/pdal/Polygon.cpp.o CMakeFiles/pdalcpp.dir/pdal/QuadIndex.cpp.o CMakeFiles/pdalcpp.dir/pdal/Reader.cpp.o CMakeFiles/pdalcpp.dir/pdal/Scaling.cpp.o CMakeFiles/pdalcpp.dir/pdal/SpatialReference.cpp.o CMakeFiles/pdalcpp.dir/pdal/SrsBounds.cpp.o CMakeFiles/pdalcpp.dir/pdal/Stage.cpp.o CMakeFiles/pdalcpp.dir/pdal/StageExtensions.cpp.o CMakeFiles/pdalcpp.dir/pdal/StageFactory.cpp.o CMakeFiles/pdalcpp.dir/pdal/Streamable.cpp.o CMakeFiles/pdalcpp.dir/pdal/SubcommandKernel.cpp.o CMakeFiles/pdalcpp.dir/pdal/Writer.cpp.o CMakeFiles/pdalcpp.dir/pdal/XMLSchema.cpp.o CMakeFiles/pdalcpp.dir/pdal/compression/DeflateCompression.cpp.o CMakeFiles/pdalcpp.dir/pdal/compression/LazPerfVlrCompression.cpp.o CMakeFiles/pdalcpp.dir/pdal/pdal_config.cpp.o CMakeFiles/pdalcpp.dir/filters/private/DimRange.cpp.o CMakeFiles/pdalcpp.dir/filters/private/Point.cpp.o CMakeFiles/pdalcpp.dir/filters/private/Segmentation.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/CSF.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/Cloth.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/Constraint.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/Particle.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/Rasterization.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/XYZReader.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/c2cdist.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/point_cloud.cpp.o CMakeFiles/pdalcpp.dir/filters/private/delaunator.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/AssignParser.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/AssignStatement.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/BaseParser.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/ConditionalExpression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/ConditionalParser.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/Expression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/IdentExpression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/Lexer.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/MathExpression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/MathParser.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/HexGrid.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/Hexagon.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/Path.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/Processor.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/Segment.cpp.o CMakeFiles/pdalcpp.dir/filters/private/mongoexpression/Comparison.cpp.o CMakeFiles/pdalcpp.dir/filters/private/mongoexpression/Expression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/mongoexpression/LogicGate.cpp.o CMakeFiles/pdalcpp.dir/io/private/GDALGrid.cpp.o CMakeFiles/pdalcpp.dir/io/private/copc/Connector.cpp.o CMakeFiles/pdalcpp.dir/io/private/copc/Entry.cpp.o CMakeFiles/pdalcpp.dir/io/private/copc/Tile.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/BuPyramid.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/Grid.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/Output.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/Processor.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/PyramidManager.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/Reprocessor.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/Addon.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/Connector.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/EptInfo.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/FixedPointLayout.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/TileContents.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Geotiff.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Header.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Srs.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Summary.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Utils.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Vlr.cpp.o CMakeFiles/pdalcpp.dir/kernels/private/density/OGR.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/MathUtils.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/Raster.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/SrsTransform.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/StageRunner.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/BaseBand.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/ErrorHandler.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/GDALUtils.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/Raster.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/SpatialRef.cpp.o  -Wl,-rpath,/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/lib  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgdal.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgeotiff.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libxml2.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libiconv.tbd  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libcharset.tbd  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  -lm  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  -Wl,-reexport_library,/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/lib/libpdal_util.14.3.0.dylib  lib/libpdal_util.14.3.0.dylib  lib/libpdal_arbiter.a  lib/libpdal_kazhdan.a  lib/libpdal_lazperf.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjson-c.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libxml2.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libzstd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgeos_c.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgeos.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libproj.a  -lm  -ldl  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsqlite3.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libqhullstatic_rd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libexpat.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libtiffd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpeg.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpng16d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libLerc.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libgif.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libnetcdf.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcurl-d.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libm.tbd  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libhdf5_hl_debug.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libhdf5_debug.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libszip_debug.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebp.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsharpyuv.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsqlite3.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpq.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libopenjp2.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libkmlengine.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libkmldom.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libkmlbase.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libminizip.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libexpat.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liburiparser.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libexpat.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liburiparser.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libminizip.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liburiparser.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpcre2-8.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libspatialite.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpq.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libssl.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcrypto.a  -ldl  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpgport.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpgcommon.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libiconv.tbd  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libcharset.tbd  -lm  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcurl-d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  -framework SystemConfiguration  -framework Security  -framework CoreFoundation  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libssl.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcrypto.a && :
FAILED: lib/libpdalcpp.14.3.0.dylib 
: && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -dynamiclib -Wl,-headerpad_max_install_names -compatibility_version 14.0.0 -current_version 14.3.0 -o lib/libpdalcpp.14.3.0.dylib -install_name @rpath/libpdalcpp.14.dylib CMakeFiles/pdalcpp.dir/filters/ApproximateCoplanarFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/AssignFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/CSFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ChipperFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ClusterFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ColorinterpFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ColorizationFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/CovarianceFeaturesFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/CropFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DBSCANFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DEMFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DecimationFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DelaunayFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/DividerFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ELMFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/EigenvaluesFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/EstimateRankFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/FaceRasterFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/FarthestPointSamplingFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/FerryFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/GpsTimeConvert.cpp.o CMakeFiles/pdalcpp.dir/filters/GreedyProjection.cpp.o CMakeFiles/pdalcpp.dir/filters/GroupByFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HagDelaunayFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HagDemFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HagNnFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HeadFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/HexBinFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/IQRFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/InfoFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/IterativeClosestPoint.cpp.o CMakeFiles/pdalcpp.dir/filters/LOFFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/LiTreeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/LloydKMeansFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/LocateFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MADFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MergeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MiniballFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MongoExpressionFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/MortonOrderFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/NNDistanceFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/NeighborClassifierFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/NormalFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/OptimalNeighborhoodFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/OutlierFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/OverlayFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/PMFFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/PlaneFitFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/PoissonFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ProjPipelineFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/RadialDensityFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/RandomizeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/RangeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ReciprocityFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/RelaxationDartThrowing.cpp.o CMakeFiles/pdalcpp.dir/filters/ReprojectionFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ReturnsFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SMRFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SampleFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SeparateScanLineFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ShellFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SkewnessBalancingFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SortFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/SplitterFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/StatsFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/StreamCallbackFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/TailFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/TransformationFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/VoxelCenterNearestNeighborFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/VoxelCentroidNearestNeighborFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/VoxelDownsizeFilter.cpp.o CMakeFiles/pdalcpp.dir/filters/ZsmoothFilter.cpp.o CMakeFiles/pdalcpp.dir/io/BpfCompressor.cpp.o CMakeFiles/pdalcpp.dir/io/BpfHeader.cpp.o CMakeFiles/pdalcpp.dir/io/BpfReader.cpp.o CMakeFiles/pdalcpp.dir/io/BpfWriter.cpp.o CMakeFiles/pdalcpp.dir/io/CopcReader.cpp.o CMakeFiles/pdalcpp.dir/io/CopcWriter.cpp.o CMakeFiles/pdalcpp.dir/io/EptAddonWriter.cpp.o CMakeFiles/pdalcpp.dir/io/EptReader.cpp.o CMakeFiles/pdalcpp.dir/io/FauxReader.cpp.o CMakeFiles/pdalcpp.dir/io/GDALReader.cpp.o CMakeFiles/pdalcpp.dir/io/GDALWriter.cpp.o CMakeFiles/pdalcpp.dir/io/GltfWriter.cpp.o CMakeFiles/pdalcpp.dir/io/Ilvis2MetadataReader.cpp.o CMakeFiles/pdalcpp.dir/io/Ilvis2Reader.cpp.o CMakeFiles/pdalcpp.dir/io/LasHeader.cpp.o CMakeFiles/pdalcpp.dir/io/LasReader.cpp.o CMakeFiles/pdalcpp.dir/io/LasVLR.cpp.o CMakeFiles/pdalcpp.dir/io/LasWriter.cpp.o CMakeFiles/pdalcpp.dir/io/MemoryViewReader.cpp.o CMakeFiles/pdalcpp.dir/io/NullWriter.cpp.o CMakeFiles/pdalcpp.dir/io/OGRWriter.cpp.o CMakeFiles/pdalcpp.dir/io/ObjReader.cpp.o CMakeFiles/pdalcpp.dir/io/OptechReader.cpp.o CMakeFiles/pdalcpp.dir/io/PcdHeader.cpp.o CMakeFiles/pdalcpp.dir/io/PcdReader.cpp.o CMakeFiles/pdalcpp.dir/io/PcdWriter.cpp.o CMakeFiles/pdalcpp.dir/io/PlyReader.cpp.o CMakeFiles/pdalcpp.dir/io/PlyWriter.cpp.o CMakeFiles/pdalcpp.dir/io/PtsReader.cpp.o CMakeFiles/pdalcpp.dir/io/QfitReader.cpp.o CMakeFiles/pdalcpp.dir/io/RasterWriter.cpp.o CMakeFiles/pdalcpp.dir/io/SbetReader.cpp.o CMakeFiles/pdalcpp.dir/io/SbetSmrmsgReader.cpp.o CMakeFiles/pdalcpp.dir/io/SbetWriter.cpp.o CMakeFiles/pdalcpp.dir/io/TIndexReader.cpp.o CMakeFiles/pdalcpp.dir/io/TerrasolidReader.cpp.o CMakeFiles/pdalcpp.dir/io/TextReader.cpp.o CMakeFiles/pdalcpp.dir/io/TextWriter.cpp.o CMakeFiles/pdalcpp.dir/kernels/ChamferKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/DeltaKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/DensityKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/EvalKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/GroundKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/HausdorffKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/InfoKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/MergeKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/PipelineKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/RandomKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/SortKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/SplitKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/TIndexKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/TileKernel.cpp.o CMakeFiles/pdalcpp.dir/kernels/TranslateKernel.cpp.o CMakeFiles/pdalcpp.dir/pdal/ColumnPointTable.cpp.o CMakeFiles/pdalcpp.dir/pdal/DbReader.cpp.o CMakeFiles/pdalcpp.dir/pdal/DbWriter.cpp.o CMakeFiles/pdalcpp.dir/pdal/DynamicLibrary.cpp.o CMakeFiles/pdalcpp.dir/pdal/Filter.cpp.o CMakeFiles/pdalcpp.dir/pdal/Geometry.cpp.o CMakeFiles/pdalcpp.dir/pdal/KDIndex.cpp.o CMakeFiles/pdalcpp.dir/pdal/Kernel.cpp.o CMakeFiles/pdalcpp.dir/pdal/Log.cpp.o CMakeFiles/pdalcpp.dir/pdal/Metadata.cpp.o CMakeFiles/pdalcpp.dir/pdal/Options.cpp.o CMakeFiles/pdalcpp.dir/pdal/PDALUtils.cpp.o CMakeFiles/pdalcpp.dir/pdal/PipelineExecutor.cpp.o CMakeFiles/pdalcpp.dir/pdal/PipelineManager.cpp.o CMakeFiles/pdalcpp.dir/pdal/PipelineReaderJSON.cpp.o CMakeFiles/pdalcpp.dir/pdal/PipelineWriter.cpp.o CMakeFiles/pdalcpp.dir/pdal/PluginDirectory.cpp.o CMakeFiles/pdalcpp.dir/pdal/PluginManager.cpp.o CMakeFiles/pdalcpp.dir/pdal/PointLayout.cpp.o CMakeFiles/pdalcpp.dir/pdal/PointTable.cpp.o CMakeFiles/pdalcpp.dir/pdal/PointView.cpp.o CMakeFiles/pdalcpp.dir/pdal/Polygon.cpp.o CMakeFiles/pdalcpp.dir/pdal/QuadIndex.cpp.o CMakeFiles/pdalcpp.dir/pdal/Reader.cpp.o CMakeFiles/pdalcpp.dir/pdal/Scaling.cpp.o CMakeFiles/pdalcpp.dir/pdal/SpatialReference.cpp.o CMakeFiles/pdalcpp.dir/pdal/SrsBounds.cpp.o CMakeFiles/pdalcpp.dir/pdal/Stage.cpp.o CMakeFiles/pdalcpp.dir/pdal/StageExtensions.cpp.o CMakeFiles/pdalcpp.dir/pdal/StageFactory.cpp.o CMakeFiles/pdalcpp.dir/pdal/Streamable.cpp.o CMakeFiles/pdalcpp.dir/pdal/SubcommandKernel.cpp.o CMakeFiles/pdalcpp.dir/pdal/Writer.cpp.o CMakeFiles/pdalcpp.dir/pdal/XMLSchema.cpp.o CMakeFiles/pdalcpp.dir/pdal/compression/DeflateCompression.cpp.o CMakeFiles/pdalcpp.dir/pdal/compression/LazPerfVlrCompression.cpp.o CMakeFiles/pdalcpp.dir/pdal/pdal_config.cpp.o CMakeFiles/pdalcpp.dir/filters/private/DimRange.cpp.o CMakeFiles/pdalcpp.dir/filters/private/Point.cpp.o CMakeFiles/pdalcpp.dir/filters/private/Segmentation.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/CSF.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/Cloth.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/Constraint.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/Particle.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/Rasterization.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/XYZReader.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/c2cdist.cpp.o CMakeFiles/pdalcpp.dir/filters/private/csf/point_cloud.cpp.o CMakeFiles/pdalcpp.dir/filters/private/delaunator.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/AssignParser.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/AssignStatement.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/BaseParser.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/ConditionalExpression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/ConditionalParser.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/Expression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/IdentExpression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/Lexer.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/MathExpression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/expr/MathParser.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/HexGrid.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/Hexagon.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/Path.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/Processor.cpp.o CMakeFiles/pdalcpp.dir/filters/private/hexer/Segment.cpp.o CMakeFiles/pdalcpp.dir/filters/private/mongoexpression/Comparison.cpp.o CMakeFiles/pdalcpp.dir/filters/private/mongoexpression/Expression.cpp.o CMakeFiles/pdalcpp.dir/filters/private/mongoexpression/LogicGate.cpp.o CMakeFiles/pdalcpp.dir/io/private/GDALGrid.cpp.o CMakeFiles/pdalcpp.dir/io/private/copc/Connector.cpp.o CMakeFiles/pdalcpp.dir/io/private/copc/Entry.cpp.o CMakeFiles/pdalcpp.dir/io/private/copc/Tile.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/BuPyramid.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/Grid.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/Output.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/Processor.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/PyramidManager.cpp.o CMakeFiles/pdalcpp.dir/io/private/copcwriter/Reprocessor.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/Addon.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/Connector.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/EptInfo.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/FixedPointLayout.cpp.o CMakeFiles/pdalcpp.dir/io/private/ept/TileContents.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Geotiff.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Header.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Srs.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Summary.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Utils.cpp.o CMakeFiles/pdalcpp.dir/io/private/las/Vlr.cpp.o CMakeFiles/pdalcpp.dir/kernels/private/density/OGR.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/MathUtils.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/Raster.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/SrsTransform.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/StageRunner.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/BaseBand.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/ErrorHandler.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/GDALUtils.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/Raster.cpp.o CMakeFiles/pdalcpp.dir/pdal/private/gdal/SpatialRef.cpp.o  -Wl,-rpath,/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/lib  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgdal.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgeotiff.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libxml2.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libiconv.tbd  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libcharset.tbd  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  -lm  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  -Wl,-reexport_library,/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/pdal/arm64-osx-dbg/lib/libpdal_util.14.3.0.dylib  lib/libpdal_util.14.3.0.dylib  lib/libpdal_arbiter.a  lib/libpdal_kazhdan.a  lib/libpdal_lazperf.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjson-c.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libxml2.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libzstd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgeos_c.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgeos.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libproj.a  -lm  -ldl  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsqlite3.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libqhullstatic_rd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libexpat.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libtiffd.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liblzma.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpeg.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpng16d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libLerc.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libgif.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libnetcdf.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcurl-d.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libm.tbd  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libhdf5_hl_debug.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libhdf5_debug.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libszip_debug.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libwebp.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsharpyuv.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libsqlite3.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpq.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libopenjp2.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libkmlengine.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libkmldom.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libkmlbase.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libminizip.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libexpat.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liburiparser.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libexpat.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liburiparser.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libminizip.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/liburiparser.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpcre2-8.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libspatialite.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpq.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libssl.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcrypto.a  -ldl  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpgport.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpgcommon.a  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libiconv.tbd  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib/libcharset.tbd  -lm  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcurl-d.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a  -framework SystemConfiguration  -framework Security  -framework CoreFoundation  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libssl.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcrypto.a && :
Undefined symbols for architecture arm64:
  "_freexl_close", referenced from:
      _load_XL in libspatialite.a(shapefiles.o)
      _vXL_create in libspatialite.a(libsplite_la-virtualXL.o)
      _vXL_disconnect in libspatialite.a(libsplite_la-virtualXL.o)
  "_freexl_get_cell_value", referenced from:
      _load_XL in libspatialite.a(shapefiles.o)
      _vXL_create in libspatialite.a(libsplite_la-virtualXL.o)
      _vXL_column in libspatialite.a(libsplite_la-virtualXL.o)
      _vXL_eval_constraints in libspatialite.a(libsplite_la-virtualXL.o)
  "_freexl_get_info", referenced from:
      _load_XL in libspatialite.a(shapefiles.o)
      _vXL_create in libspatialite.a(libsplite_la-virtualXL.o)
  "_freexl_open", referenced from:
      _load_XL in libspatialite.a(shapefiles.o)
      _vXL_create in libspatialite.a(libsplite_la-virtualXL.o)
  "_freexl_select_active_worksheet", referenced from:
      _load_XL in libspatialite.a(shapefiles.o)
      _vXL_create in libspatialite.a(libsplite_la-virtualXL.o)
  "_freexl_version", referenced from:
      _fnct_freexl_version in libspatialite.a(libsplite_la-spatialite.o)
  "_freexl_worksheet_dimensions", referenced from:
      _load_XL in libspatialite.a(shapefiles.o)
      _vXL_create in libspatialite.a(libsplite_la-virtualXL.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "mygui",
      "default-features": false,
      "features": [
        "tools"
      ]
    }
  ]
}

```
</details>

--- a/ports/pdal/portfile.cmake
+++ b/ports/pdal/portfile.cmake
@@ -35,10 +35,6 @@ file(APPEND "${SOURCE_PATH}/pdal/JsonFwd.hpp" "namespace NL = nlohmann;\n")
 
 unset(ENV{OSGEO4W_HOME})
 
-if("laszip" IN_LIST FEATURES)
-    message(WARNING "The 'laszip' feature is obsolete and will be removed in the future.")
-endif()
-
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
         draco       BUILD_PLUGIN_DRACO
@@ -49,10 +45,7 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
         pgpointcloud BUILD_PLUGIN_PGPOINTCLOUD
         zstd        WITH_ZSTD
 )
-if(BUILD_PLUGIN_DRACO)
-    vcpkg_find_acquire_program(PKGCONFIG)
-endif()
-
+vcpkg_find_acquire_program(PKGCONFIG)
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS

--- a/ports/pdal/vcpkg.json
+++ b/ports/pdal/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "pdal",
   "version": "2.4.3",
-  "port-version": 2,
+  "port-version": 3,
   "description": "PDAL - Point Data Abstraction Library is a library for manipulating point cloud data.",
   "homepage": "https://pdal.io/",
   "license": null,
@@ -63,9 +63,6 @@
       "dependencies": [
         "zlib"
       ]
-    },
-    "laszip": {
-      "description": "Obsolete. PDAL includes LAZperf."
     },
     "liblzma": {
       "description": "Support for compression/decompression with LZMA",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6066,7 +6066,7 @@
     },
     "pdal": {
       "baseline": "2.4.3",
-      "port-version": 2
+      "port-version": 3
     },
     "pdal-c": {
       "baseline": "2.1",

--- a/versions/p-/pdal.json
+++ b/versions/p-/pdal.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "e0e82c8480da2e69cc7532911895844095466aad",
+      "version": "2.4.3",
+      "port-version": 3
+    },
+    {
       "git-tree": "ecc35d5b8ff83ebb84e0f2edfa7c5fef7630467f",
       "version": "2.4.3",
       "port-version": 2

